packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -62,6 +62,9 @@ class AccountSignup extends React.Component {
         // Handle lower level modal controls due to overriding modal rendering
         this.props.isModalVisible(false);
         this.props.enableApp();
+        if (this.props.is_europe) {
+            this.props.toggleAccountTypesModal(true);
```

Comment:
```suggestion
this.props.toggleAccountTypesModal(this.props.is_europe);
```
Can we do this instead to remove indentation?

Reply:
It is neater as coding style, but when the account is not for EU, we are calling an extra action. but I think generally it is a good idea to remove that if